crates/op-rbuilder/src/builders/flashblocks/payload.rs

```diff
@@ -493,33 +511,57 @@ where
             };
             let _entered = fb_span.enter();
 
+            if ctx.flashblock_index() > ctx.target_flashblock_count() {
```

Comment:
We need to check that it's okay. AFAIK if future is completed before we get engine_getPayload it won't return result to it

Reply:
i tested this by removing `--flashblocks.enabled` for rollup-boost in builder-playground and running with telemetry, everything appears to be the same as before, what should i be looking for specifically to confirm?